ui/src/components/Dashboard/index.jsx

```diff
@@ -286,22 +286,18 @@ export default class Dashboard extends Component {
               </IconButton>
             </Tooltip>
             <Tooltip placement="bottom" title="Documentation">
-              <IconButton
-                className={classes.appBarButton}
-                component={Link}
-                to={DOCS_PATH_PREFIX}>
+              <IconButton component={Link} to={DOCS_PATH_PREFIX}>
                 <BookOpenPageVariantIcon className={classes.appIcon} />
               </IconButton>
             </Tooltip>
             {helpView && (
               <Tooltip placement="bottom" title="Page Information">
-                <IconButton
-                  onClick={this.handleHelpViewToggle}
-                  className={classes.appBarButton}>
+                <IconButton onClick={this.handleHelpViewToggle}>
                   <HelpIcon className={classes.appIcon} />
                 </IconButton>
               </Tooltip>
             )}
+            <UserMenu navOpen appBar />
```

Comment:
Interesting. I thought we had removed `navOpen`?

Reply:
This problem occur when I have done rebase. It automatically got resolve adding this.

Reply:
No, navOpen is being used in the code. see the below comment I put the reference of the code.

Reply:
Going to investigate and if it is not needed I will remove it.

Reply:
Remove `navOpen`.